q2_moshpit/plugin_setup.py

```diff
@@ -1460,3 +1465,41 @@
     ReferenceDB[BuscoDB], BuscoDatabaseDirFmt
 )
 importlib.import_module('q2_moshpit.busco.types._transformer')
+
+plugin.methods.register_function(
+    function=q2_moshpit.eggnog.fetch_eggnog_hmmer_db,
+    inputs={},
+    parameters={
+        "taxon_id": Int % Range(2, None)
+    },
+    parameter_descriptions={
+        "taxon_id": "Taxon ID number."
+    },
+    outputs=[
+        ("idmap", EggnogHmmerIdmap % Properties("eggnog")),
+        ("hmm_db", ProfileHMM[MultipleProtein] % Properties("eggnog")),
+        ("pressed_hmm_db", ProfileHMM[PressedProtein] % Properties("eggnog")),
+        ("seed-alignments", GenomeData[Proteins] % Properties("eggnog"))
```

Comment:
Hmmm, looks like we forgot to test the output name change? :P
```suggestion
        ("seed_alignments", GenomeData[Proteins] % Properties("eggnog"))
```